les/server_handler.go

```diff
@@ -948,3 +950,7 @@ func (h *serverHandler) broadcastHeaders() {
 		}
 	}
 }
+
+func (h *serverHandler) talkRequestHandler(id enode.ID, addr *net.UDPAddr, payload []byte) ([]byte, bool) {
```

Comment:
Does it mean servers can only receive the UDP packet and then reply? 
So in the following scenarios:

* Client should ask the current token price
* Client should ask the current balance to determine whether they need to buy more tokens for stable connection